modules/cli-options/src/main/scala/scala/cli/commands/ScalaNativeOptions.scala

```diff
@@ -43,9 +43,14 @@ final case class ScalaNativeOptions(
   @Group("Scala Native")
   @Hidden
   @HelpMessage("Use default compile options")
-    nativeCompileDefaults: Option[Boolean] = None //TODO does it even work when we default it to true while handling?
+    nativeCompileDefaults: Option[Boolean] = None, //TODO does it even work when we default it to true while handling?
+
+  @Group("Scala Native")
+  @HelpMessage("Do not embed resources into the Scala Native binary (java style Resources will not be able to be used)")
+    noEmbed: Option[Boolean] = None
```

Comment:
No need to use a negation in the name here IMO.
```suggestion
    embedResources: Option[Boolean] = None
```
Users can pass `--embed-resources=false` to negate the flag.

Reply:
Embedding is meant to be enabled by default and I feel that having "--embed-resources" do nothing and asking users to --embed-resources=false to have any kind of effect would be unintuitive. Is that a problem? wdyt

Reply:
I hold by `--embed-resources=false`.  IIRC there are no negative CLI options such as this in Scala CLI. For example, users can pass `--server=false` to disable using Bloop. Better stick to that convention for now.

To give you some more context, I added a `--no-default` option in coursier [quite some time ago](https://github.com/coursier/coursier/commit/3b4b773c6433de070d19239b1ad182037e5b413c), and I still regret it up to this day…

Reply:
Makes sense, I honestly forgot about `--server=false` when I wrote the original comment